### ISSUE TITLE
"The conditional check 'not logging_elasticsearch_rollout_override | bool' failed. The error was: error while evaluating conditional (not logging_elasticsearch_rollout_override | bool)

### ISSUE THREAD

**Issue by SubhakarKotta** [openshift/openshift-ansible#9100]
#### Description
Getting below error while installing openshift 3.9 centos

FAILED! => {"msg": "The conditional check 'not logging_elasticsearch_rollout_override | bool' failed. The error was: error while evaluating conditional (not logging_elasticsearch_rollout_override | bool): 'logging_elasticsearch_rollout_override' is undefined\n\nThe error appears to have been in '/root/Downloads/installcentos-master/openshift-ansible/roles/openshift_logging_elasticsearch/handlers/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: \"Restarting logging-{{ _cluster_component }} cluster\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes.  Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

##### Version

openshift-ansible : release-3.9

ansible 2.6.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr 11 2018, 07:36:10) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

Docker version 1.13.1, build 94f4240/1.13.1

**Comment by ewolinetz** [comment]
@mtnbikenc did the way this was evaluated as `bool` change in Ansible? I didn't see this with ansible 5.5.0 and we see this passing for CI tests too

It wouldn't be difficult to add a `default(false)` to that when condition, but I'm curious as to why this is now having an error.

**Comment by alejojo** [comment]
@SubhakarKotta  I had same issue. If you need quick fix then install 2.5.6 from ansible repositories https://releases.ansible.com/ansible/rpm/release/epel-7-x86_64/

**Comment by SubhakarKotta** [comment]
1 week back it was installed with out any issues.Suddenly it was not working now. I tried on complete new setup and it doesnt work.

**Comment by ewolinetz** [comment]
@SubhakarKotta Can you try rolling back your version of Ansible? I tested this with Ansible version 2.5.5 yesterday and did not receive this error

**Comment by dd011** [comment]
@ewolinetz i merger #9145 modification but met this error in ansible 2.6:
```
Deployment failed. Correlation ID: 144eb6fd-615f-4c5e-9999-88dedc5bb70d. {
  "status": "Failed",
  "error": {
    "code": "ResourceDeploymentFailure",
    "message": "The resource operation completed with terminal provisioning state 'Failed'.",
    "details": [
      {
        "code": "VMExtensionProvisioningError",
        "message": "VM has reported a failure when processing extension 'deployOpenShift'. Error message: \"Enable failed: processing file downloads failed: failed to download file[0]: failed to download file: unexpected status code: got=404 expected=200\"."
      }
    ]
  }
}
```
do you meet this problem?

**Comment by ewolinetz** [comment]
Should be resolved with #9145
